dev-resources/private/test/src/cljs/foo/load_require.cljs

```diff
@@ -0,0 +1,6 @@
+(ns foo.load-require)
+
+;;; a repl special
+(require 'foo.bar.baz)
+
```

Comment:
let's change and test that `(def b (foo.bar.baz/mul 4 5))` is equal to 20 in `foo.load-require`